index.js

```diff
@@ -132,7 +133,12 @@ async function lint(
       if (config.isAbsoluteURL(ruleset)) {
         rulesetPath = ruleset
       } else {
-        rulesetPath = path.resolve(targetDir, ruleset)
+        if (fs.existsSync(path.resolve(targetDir, ruleset))) {
+          rulesetPath = path.resolve(targetDir, ruleset)
+        }
```

Comment:
Should this be an 'else'?

In the conversation you said:  "While maintaining the current implementation, the rulesetPath logic will try to use the context path if target repo path doesn't exist."